src/prefect/cli/cloud.py

```diff
@@ -96,7 +97,7 @@ def receive_failure(payload: LoginFailed):
 
 
 async def serve_login_api(cancel_scope):
-    config = uvicorn.Config(login_api, port=3001, log_level="debug")
+    config = uvicorn.Config(login_api, port=3001, log_level="critical")
```

Comment:
is there any risk of sensitive data being logged by uvicorn?

Reply:
I do not think so even with debug level logs. This is silencing their logger.